build/nuget/build.ps1

```diff
@@ -21,7 +21,7 @@ if(!$Repo) {
 
 $version = "0.0.1"
```

Comment:
This is not new change, but we are currently 0.0.2 according to build\winobjc.version.